test/lib/docker.spec.ts

```diff
@@ -51,7 +51,7 @@ describe("docker", () => {
       const result = docker.save(image, destination);
 
       //  rejects with expected error
-      await expect(result).rejects.toThrowError("server error");
+      await expect(result).rejects.toThrowError();
```

Comment:
Different Docker engines are returning different error strings, for now we will just expect an exception to be thrown and the Promise rejected